client/state/action-types.js

```diff
@@ -268,6 +268,9 @@ export const SUPPORT_USER_ACTIVATE = 'SUPPORT_USER_ACTIVATE';
 export const SUPPORT_USER_ERROR = 'SUPPORT_USER_ERROR';
 export const SUPPORT_USER_TOGGLE_DIALOG = 'SUPPORT_USER_TOGGLE_DIALOG';
 export const SUPPORT_USER_TOKEN_FETCH = 'SUPPORT_USER_TOKEN_FETCH';
+export const TERMS_ADD_REQUEST = 'TERMS_ADD_REQUEST';
```

Comment:
In #6179, I originally named the save actions `POST_SAVE_REQUEST`, but later simplified it to `POST_SAVE`. Do you think there's value in the `_REQUEST` suffix? I think there's an argument to be made about clarifying that it's issuing a network request.